scripts/ci-check-to-ensure-sql-is-in-lowercase.mjs

```diff
@@ -0,0 +1,98 @@
+import fs from "fs";
+import path from "path";
+
+const regex = /```sql\n([\s\S]*?)\n```/g;
+const keywords = [
+  "SELECT",
+  "UPDATE",
+  "INSERT",
+  "DELETE",
+  "WITH",
+  "FROM",
+  "WHERE",
+  "ORDER BY",
+  "GROUP BY",
+  "HAVING",
+  "AS",
+  "VALUES",
+];
+
+const WEBSITE_FOLDERS = [
+  "../postgraphile/website",
+  "../grafast/website",
+  "../graphile-build/website",
+  "../utils/website",
```

Comment:
We want this script to work no matter where you run it from, so let's make these absolute:

```suggestion
  `${__dirname}/../postgraphile/website`,
  `${__dirname}/../grafast/website`,
  `${__dirname}/../graphile-build/website`,
  `${__dirname}/../utils/website`,
```